doc/en/user/source/production/java.rst

```diff
@@ -196,3 +223,16 @@ Installing native JAI manually
 You can install the native JAI manually if you encounter problems using the above installers, or if you wish to install the native JAI for more than one JDK/JRE.
 
 Please refer to the `GeoTools page on JAI installation <http://docs.geotools.org/latest/userguide/build/install/jdk.html#java-extensions-optional>`_ for details.
+
+Running on Java 11
+------------------
```

Comment:
New heading for Java 11, provides a summary of options, this way it is very clear Java 11 works out of the box and does not have as many optional installs.